packages/form-footer/src/PrimaryButton.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Button from '@leafygreen-ui/button';
+import { once } from 'lodash';
```

Comment:
There's a helper for this inside of `lib`

```
export const consoleOnce = {
  error: once(console.error),
  warn: once(console.warn),
  log: once(console.log),
};
```

Reply:
Didn't know this! Thanks! :)